redis-om-spring/src/main/java/com/redis/om/spring/convert/BytesToUlidConverter.java

```diff
@@ -0,0 +1,27 @@
+package com.redis.om.spring.convert;
+
+import java.nio.charset.StandardCharsets;
+
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.data.convert.ReadingConverter;
+import org.springframework.util.ObjectUtils;
+
+import com.github.f4b6a3.ulid.Ulid;
```

Comment:
@bsbodden pardon my ignorance here! I found a lot of use of [sulky ulid](https://mvnrepository.com/artifact/de.huxhorn.sulky/de.huxhorn.sulky.ulid) all over RedHat, Netflix, and others. Is one better than the other?

Not that it's worth anything, but that library is also far higher up on maven.org

Reply:
We can evaluate and swap if needed. Let's create an issue to follow up.

Reply:
See https://github.com/redis/redis-om-spring/issues/87